src/server/config.rs

```diff
@@ -97,6 +98,7 @@ impl Default for Config {
             end_point_recursion_limit: 1000,
             end_point_batch_row_limit: DEFAULT_ENDPOINT_BATCH_ROW_LIMIT,
             snap_max_write_bytes_per_sec: ReadableSize(DEFAULT_SNAP_MAX_BYTES_PER_SEC),
+            snap_max_total_size: ReadableSize(0),
```

Comment:
0? unlimited snapshot size by default?

Reply:
Yes, for general users, snapshot gc strategy is keep them 4 hours at most, if they are not expired.